chemistry/topologyobjects.py

```diff
@@ -2155,7 +2155,7 @@ def __eq__(self, other):
 
     def __repr__(self):
         return ('<%s; phi_k=%.3f, per=%d, phase=%.3f, scee=%.3f, scnb=%.3f>' %
-                (type(self).__name__, self.phi_k, self.per, self.phase,
+                (type(self).__name__, self.phi_k, self.per, _strip_units(self.phase, u.degree),
```

Comment:
This should've been done in the constructor (my bad), but I'll fix it post-merge.